### ISSUE TITLE
GraphQL: Asset inside Replicator fails

### ISSUE THREAD

**Issue by St3Ko** [statamic/cms#3200]
## Bug Description
when i try to query an asset field inside a replicator the query fails with an "Internal server error"

```
"Argument 1 passed to Statamic\\GraphQL\\Types\\AssetInterface::resolveType() must implement interface Statamic\\Contracts\\Assets\\Asset, instance of Statamic\\Fields\\Value given"
```

this works:
```graphql
query {
  entry(id: "home") {
    ... on Entry_Pages_Pages {
      image {
        url
      }
    }
  }
}
```

this not:
```graphql
query {
  entry(id: "home") {
    ... on Entry_Pages_Pages {
      image {
        url
      }
      page_blocks {
        ... on Set_PageBlocks_Media {
          image {
            url
          }
        }
      }
    }
  }
}
```

## How to Reproduce
create a replicator with a set that has an asset in it and try to query it

## Extra Detail
```json
{
  "errors": [
    {
      "debugMessage": "Argument 1 passed to Statamic\\GraphQL\\Types\\AssetInterface::resolveType() must implement interface Statamic\\Contracts\\Assets\\Asset, instance of Statamic\\Fields\\Value given",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "locations": [
        {
          "line": 23,
          "column": 11
        }
      ],
      "path": [
        "entry",
        "page_blocks",
        0,
        "image"
      ],
      "trace": [
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/rebing/graphql-laravel/src/Support/InterfaceType.php",
          "line": 24,
          "function": "call_user_func_array(array(2), array(3))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Type/Definition/InterfaceType.php",
          "line": 174,
          "call": "Rebing\\GraphQL\\Support\\InterfaceType::Rebing\\GraphQL\\Support\\{closure}(instance of Statamic\\Fields\\Value, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 941,
          "call": "GraphQL\\Type\\Definition\\InterfaceType::resolveType(instance of Statamic\\Fields\\Value, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 790,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: AssetInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(4), instance of Statamic\\Fields\\Value)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: AssetInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(4), instance of Statamic\\Fields\\Value)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: AssetInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(4), instance of Statamic\\Fields\\Value)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1196,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Set_PageBlocks_Media, array(2), instance of ArrayObject(1), array(4))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1146,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Set_PageBlocks_Media, array(2), array(3), instance of ArrayObject(1))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1107,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: Set_PageBlocks_Media, instance of ArrayObject(1), array(3), array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 975,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: Set_PageBlocks_Media, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 790,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: Sets_PageBlocks, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: Sets_PageBlocks, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 888,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: Sets_PageBlocks, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 762,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeListValue(GraphQLType: Sets_PageBlocks, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(1))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: Sets_PageBlocks, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(1))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: Sets_PageBlocks, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(1))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1196,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Entry_Pages_Pages, instance of Statamic\\Entries\\Entry, instance of ArrayObject(1), array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1146,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Entry_Pages_Pages, instance of Statamic\\Entries\\Entry, array(1), instance of ArrayObject(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1107,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: Entry_Pages_Pages, instance of ArrayObject(1), array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 975,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: Entry_Pages_Pages, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 790,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 655,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1196,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Query, null, instance of ArrayObject(1), array(1))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 264,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Query, null, array(0), instance of ArrayObject(1))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 215,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeOperation(instance of GraphQL\\Language\\AST\\OperationDefinitionNode, null)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/Executor/Executor.php",
          "line": 156,
          "call": "GraphQL\\Executor\\ReferenceExecutor::doExecute()"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 162,
          "call": "GraphQL\\Executor\\Executor::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, instance of GraphQL\\Language\\AST\\DocumentNode, null, null, null, null, null)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 94,
          "call": "GraphQL\\GraphQL::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, '# Welcome to GraphiQL\n#\n# GraphiQL is an in-browser tool for writing, validating, and\n# testing GraphQL queries.\n#\n# Type queries into this side of the screen, and you will see intelligent\n# typeaheads aware of the current GraphQL type schema and live syntax and\n# validation errors highlighted within the text.\n#\n# You can use the \"Explorer\" to click and build your queries\n#\n# GraphQL queries typically start with a \"{\" character. Lines that start\n# with a # are ignored.\n\nquery {\n  entry(id: \"home\") {\n    ... on Entry_Pages_Pages {\n      image {\n        url\n      }\n      page_blocks {\n        ... on Set_PageBlocks_Media {\n          image {\n            url\n          }\n        }\n      }\n    }\n  }\n}', null, null, null, null, null, null)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/rebing/graphql-laravel/src/GraphQL.php",
          "line": 148,
          "call": "GraphQL\\GraphQL::executeQuery(instance of GraphQL\\Type\\Schema, '# Welcome to GraphiQL\n#\n# GraphiQL is an in-browser tool for writing, validating, and\n# testing GraphQL queries.\n#\n# Type queries into this side of the screen, and you will see intelligent\n# typeaheads aware of the current GraphQL type schema and live syntax and\n# validation errors highlighted within the text.\n#\n# You can use the \"Explorer\" to click and build your queries\n#\n# GraphQL queries typically start with a \"{\" character. Lines that start\n# with a # are ignored.\n\nquery {\n  entry(id: \"home\") {\n    ... on Entry_Pages_Pages {\n      image {\n        url\n      }\n      page_blocks {\n        ... on Set_PageBlocks_Media {\n          image {\n            url\n          }\n        }\n      }\n    }\n  }\n}', null, null, null, null, null)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/rebing/graphql-laravel/src/GraphQL.php",
          "line": 126,
          "call": "Rebing\\GraphQL\\GraphQL::queryAndReturnResult('# Welcome to GraphiQL\n#\n# GraphiQL is an in-browser tool for writing, validating, and\n# testing GraphQL queries.\n#\n# Type queries into this side of the screen, and you will see intelligent\n# typeaheads aware of the current GraphQL type schema and live syntax and\n# validation errors highlighted within the text.\n#\n# You can use the \"Explorer\" to click and build your queries\n#\n# GraphQL queries typically start with a \"{\" character. Lines that start\n# with a # are ignored.\n\nquery {\n  entry(id: \"home\") {\n    ... on Entry_Pages_Pages {\n      image {\n        url\n      }\n      page_blocks {\n        ... on Set_PageBlocks_Media {\n          image {\n            url\n          }\n        }\n      }\n    }\n  }\n}', null, array(3))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/rebing/graphql-laravel/src/GraphQLController.php",
          "line": 73,
          "call": "Rebing\\GraphQL\\GraphQL::query('# Welcome to GraphiQL\n#\n# GraphiQL is an in-browser tool for writing, validating, and\n# testing GraphQL queries.\n#\n# Type queries into this side of the screen, and you will see intelligent\n# typeaheads aware of the current GraphQL type schema and live syntax and\n# validation errors highlighted within the text.\n#\n# You can use the \"Explorer\" to click and build your queries\n#\n# GraphQL queries typically start with a \"{\" character. Lines that start\n# with a # are ignored.\n\nquery {\n  entry(id: \"home\") {\n    ... on Entry_Pages_Pages {\n      image {\n        url\n      }\n      page_blocks {\n        ... on Set_PageBlocks_Media {\n          image {\n            url\n          }\n        }\n      }\n    }\n  }\n}', null, array(3))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/rebing/graphql-laravel/src/GraphQLController.php",
          "line": 48,
          "call": "Rebing\\GraphQL\\GraphQLController::executeQuery('default', array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Controller.php",
          "line": 54,
          "call": "Rebing\\GraphQL\\GraphQLController::query(instance of Illuminate\\Http\\Request, 'default')"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php",
          "line": 45,
          "call": "Illuminate\\Routing\\Controller::callAction('query', array(2))"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Route.php",
          "line": 254,
          "call": "Illuminate\\Routing\\ControllerDispatcher::dispatch(instance of Illuminate\\Routing\\Route, instance of Rebing\\GraphQL\\GraphQLController, 'query')"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Route.php",
          "line": 197,
          "call": "Illuminate\\Routing\\Route::runController()"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 693,
          "call": "Illuminate\\Routing\\Route::run()"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 128,
          "call": "Illuminate\\Routing\\Router::Illuminate\\Routing\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/statamic/cms/src/GraphQL/Middleware/CacheResponse.php",
          "line": 18,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\GraphQL\\Middleware\\CacheResponse::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/statamic/cms/src/Http/Middleware/RequireStatamicPro.php",
          "line": 17,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\RequireStatamicPro::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/statamic/cms/src/Http/Middleware/SwapExceptionHandler.php",
          "line": 19,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\SwapExceptionHandler::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 103,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 695,
          "call": "Illuminate\\Pipeline\\Pipeline::then(instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 670,
          "call": "Illuminate\\Routing\\Router::runRouteWithinStack(instance of Illuminate\\Routing\\Route, instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 636,
          "call": "Illuminate\\Routing\\Router::runRoute(instance of Illuminate\\Http\\Request, instance of Illuminate\\Routing\\Route)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 625,
          "call": "Illuminate\\Routing\\Router::dispatchToRoute(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php",
          "line": 166,
          "call": "Illuminate\\Routing\\Router::dispatch(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 128,
          "call": "Illuminate\\Foundation\\Http\\Kernel::Illuminate\\Foundation\\Http\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/statamic/cms/src/Http/Middleware/CheckMultisite.php",
          "line": 14,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\CheckMultisite::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/statamic/cms/src/Http/Middleware/CheckComposerJsonScripts.php",
          "line": 14,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\CheckComposerJsonScripts::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/statamic/cms/src/Http/Middleware/PoweredByHeader.php",
          "line": 19,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\PoweredByHeader::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/barryvdh/laravel-debugbar/src/Middleware/InjectDebugbar.php",
          "line": 60,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Barryvdh\\Debugbar\\Middleware\\InjectDebugbar::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php",
          "line": 21,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php",
          "line": 21,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php",
          "line": 27,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php",
          "line": 86,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/fruitcake/laravel-cors/src/HandleCors.php",
          "line": 37,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Fruitcake\\Cors\\HandleCors::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/fideloper/proxy/src/TrustProxies.php",
          "line": 57,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Fideloper\\Proxy\\TrustProxies::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 103,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php",
          "line": 141,
          "call": "Illuminate\\Pipeline\\Pipeline::then(instance of Closure)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php",
          "line": 110,
          "call": "Illuminate\\Foundation\\Http\\Kernel::sendRequestThroughRouter(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/dev/htdocs/alpha/public/index.php",
          "line": 52,
          "call": "Illuminate\\Foundation\\Http\\Kernel::handle(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/steko/.composer/vendor/laravel/valet/server.php",
          "line": 158,
          "function": "require('/Users/steko/dev/htdocs/alpha/public/index.php')"
        }
      ]
    }
  ],
  "data": {
    "entry": {
      "image": {
        "url": "/assets/image-test.png"
      },
      "page_blocks": [
        {
          "image": null
        }
      ]
    }
  }
}
```

## Environment
Statamic 3.1.0-alpha.1 Pro
Laravel 8.26.1
PHP 7.3.21
No addons installed

**Install method:**
- Fresh install from `statamic/statamic`

**Comment by goellner** [comment]
Same error when having an image in a bard set

**Comment by jasonvarga** [comment]
Fixed for 3.1.0-alpha.2
